system/lib/standalone/standalone.c

```diff
@@ -157,19 +157,13 @@ double emscripten_get_now(void) {
 
 // C++ ABI
 
-// Emscripten disables exception catching by default, but not throwing. That
-// allows users to see a clear error if a throw happens, and 99% of the
-// overhead is in the catching, so this is a reasonable tradeoff.
-// For now, in a standalone build just terminate. TODO nice error message
-//
-// Define these symbols as weak so that when we build with exceptions
-// enabled (using wasm-eh) we get the real versions of these functions
-// as defined in libc++abi.
-
-__attribute__((__weak__))
+#if NOCATCH
```

Comment:
In other stuff we've used `EMSCRIPTEN_X` so perhaps `EMSCRIPTEN_NOCATCH`?

Reply:
Done